junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/DisplayNameUtils.java

```diff
@@ -106,12 +113,24 @@ static Supplier<String> createDisplayNameSupplierForNestedClass(Class<?> testCla
 			generator -> generator.generateDisplayNameForNestedClass(testClass));
 	}
 
+	static Supplier<String> createDisplayNameSupplierForNestedClass(List<Class<?>> enclosingInstanceTypes,
+			Class<?> testClass, JupiterConfiguration configuration) {
+		return createDisplayNameSupplier(testClass, configuration,
+			generator -> generator.generateDisplayNameForNestedClass(enclosingInstanceTypes, testClass));
+	}
+
 	private static Supplier<String> createDisplayNameSupplierForMethod(Class<?> testClass, Method testMethod,
 			JupiterConfiguration configuration) {
 		return createDisplayNameSupplier(testClass, configuration,
 			generator -> generator.generateDisplayNameForMethod(testClass, testMethod));
 	}
```

Comment:
This method and the one above for nested classes should no longer be used and therefore deleted.